crates/diesel_models/src/fraud_check.rs

```diff
@@ -25,6 +26,7 @@ pub struct FraudCheck {
     pub metadata: Option<serde_json::Value>,
     pub modified_at: PrimitiveDateTime,
     pub last_step: FraudCheckLastStep,
+    pub frm_capture_method: Option<storage_enums::CaptureMethod>, // In postFrm, we are updating capture method from automatic to manual. To store the merchant actual capture method, we are storing the actual capture method in frm_capture_method. It will be useful while approving the FRM decision.
```

Comment:
we can have a separate enum for this, just in case we might need to have different enum variants for these

Reply:
Since we are using same capture method type as payment capture method, I am just renaming `frm_capture_method` to `payment_capture_method` as we discussed.